added the FF release note for align-content in block layout (#32863)
mdn/content: files/en-us/mozilla/firefox/releases/125/index.md

### CSS

> - The {{cssxref("align-content")}} property has now been enabled to work with `display: block;` layouts. This brings all the layout positions from `flex` and `grid` to block enabling developers easily align content without convert the container to a `flex` or `grid` container simply to do so. ([Firefox bug 1882853](https://bugzil.la/1882853)).

Reviewer: ```suggestion
- The {{cssxref("align-content")}} property has been updated to work with `display: block;` layouts. This brings all the layout positions from `flex` and `grid` to `block`, enabling developers to align block-level elements without converting their container to a `flex` or `grid` container. ([Firefox bug 1882853](https://bugzil.la/1882853)).
```
